api-extractor.json

```diff
@@ -143,16 +143,22 @@
      */
     "enabled": true,
 
+    /**
+     * DEFAULT VALUE: "[ "complete" ]"
+     */
+    "reportVariants": ["complete"],
+
     /**
      * The filename for the API report files.  It will be combined with "reportFolder" or "reportTempFolder" to produce
      * a full file path.
      *
      * The file extension should be ".api.md", and the string should not contain a path separator such as "\" or "/".
+     * API Extractor will automatically append an appropriate file extension such as .api.md
```

Comment:
this a breaking change of `api-extractor`
before: `api.md`
now: `<package-name>(.beta?).api.md`